vendor/bat-native-ledger/include/bat/ledger/public/interfaces/ledger.mojom

```diff
@@ -246,6 +246,7 @@ struct TransferFee {
 };
 
 enum RewardsType {
+  UNKNOWN = -1,
   AUTO_CONTRIBUTE = 2,
```

Comment:
Not to do with this PR, but why are we bit-shifting these values?

Reply:
@tmancey legacy code 😄 but because we have this values in the db when I migrated it mojo I kept the same values.